docs/manuals/source/TasksAndConcepts/Plugins.rst

```diff
@@ -627,6 +628,13 @@ When performing restores, the plugin can do one of the following:
 * Create a new VM with new disks
 * Overwrite existing disks of an existing VM
 
+Additionally it is possible to
+
+* Skip disks by alias names using include/exclude
+* Restoring the VM only without any disks is possible by
+  * Selecting to restore only the **.ovf** file
+  * Excluding all disks by alias
```

Comment:
This Line and the one before are mit rendered correct. See https://download.bareos.org/bareos/experimental/CD/PR-402/BareosMainReference/TasksAndConcepts/Plugins.html?highlight=ovirt